### PR TITLE
Add table to list port, endpoint, framework, model, serving, and hardware for each microservice in ChatQnA

### PR DESCRIPTION
**Pull request by ctao456**
## Description

According to TCEs' feedback, add explanatory table to list port, endpoint, framework, model, serving and hardware for each microservice in ChatQnA

## Issues

`n/a`

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

`n/a`

## Tests

`n/a`

### CODE REVIEW

File: ChatQnA/README.md [hunk 1]
```diff
@@ -97,6 +97,123 @@ flowchart LR
 
 This ChatQnA use case performs RAG using LangChain, Redis VectorDB and Text Generation Inference on Intel Gaudi2 or Intel XEON Scalable Processors. The Intel Gaudi2 accelerator supports both training and inference for deep learning models in particular for LLMs. Visit [Habana AI products](https://habana.ai/products) for more details.
 
+For a full list of framework, model, serving, and hardware choices available for each of the microservice components in the ChatQnA architecture, please refer to the below table:
+
```
Comment (louie-tsai) [inline ChatQnA/README.md:101]:
```suggestion
<details>
<summary> Gaudi default compose.yaml </summary>
| MicroService    | Open Source Project                |   HW      |  Port     | Endpoint |
| --------------- | ---------------------------------- |---------- |
| Embedding     | Langchain                                  | Gaudi     | 6000 |  /v1/embaddings |
| Retriever         | Langchain                                  | Xeon      | 7000 |  /v1/retrieval         |
| Reranking       | Langchain                                  | Xeon      | 8000 | /v1/reranking        |
| LLM                |  Langchain                                  | Gaudi     | 9000 | /v1/chat/completions |
| Dataprep        |  Redis                                          | Xeon      | 6007| /v1/dataprep             |

</details>
```

**Review by louie-tsai — COMMENTED** [review]
looks too complicated and also some duplications with genAI comp README. suggest some simple version and hide it by default

File: ChatQnA/README.md (hunk 1 above)
Comment (ctao456) [reply ChatQnA/README.md]: Thank you. Revised according to comments. Included simple table in latest commit.

File: ChatQnA/README.md [hunk 2]
```diff
@@ -97,6 +97,21 @@ flowchart LR
...
+<details>
+<summary> Gaudi default compose.yaml </summary>
+
+| MicroService | Open Source Project | HW    | Port | Endpoint             |
+| ------------ | ------------------- | ----- | ---- | -------------------- |
+| Embedding    | Langchain           | Gaudi | 6000 | /v1/embaddings       |
+| Retriever    | Langchain           | Xeon  | 7000 | /v1/retrieval        |
+| Reranking    | Langchain           | Xeon  | 8000 | /v1/reranking        |
+| LLM          | Langchain           | Gaudi | 9000 | /v1/chat/completions |
+| Dataprep     | Redis               | Xeon  | 6007 | /v1/dataprep         |
```
Comment (louie-tsai) [inline ChatQnA/README.md:111]:
```suggestion
| Dataprep     | Redis, Langchain            | Xeon  | 6007 | /v1/dataprep         |
```

File: ChatQnA/README.md [hunk 3]
```diff
@@ -97,6 +97,21 @@ flowchart LR
...
+
+<details>
+<summary> Gaudi default compose.yaml </summary>
+
+| MicroService | Open Source Project | HW    | Port | Endpoint             |
+| ------------ | ------------------- | ----- | ---- | -------------------- |
+| Embedding    | Langchain           | Gaudi | 6000 | /v1/embaddings       |
+| Retriever    | Langchain           | Xeon  | 7000 | /v1/retrieval        |
+| Reranking    | Langchain           | Xeon  | 8000 | /v1/reranking        |
+| LLM          | Langchain           | Gaudi | 9000 | /v1/chat/completions |
```
Comment (louie-tsai) [inline ChatQnA/README.md:110]:
```suggestion
| LLM          | Langchain, TGI           | Gaudi | 9000 | /v1/chat/completions |
```

File: ChatQnA/README.md [hunk 4]
```diff
@@ -97,6 +97,21 @@ flowchart LR
...
+In the below, we provide a table that describes for each microservice component in the ChatQnA architecture, the default configuration of the open source project, hardware, port, and endpoint.
+
+<details>
+<summary> Gaudi default compose.yaml </summary>
+
+| MicroService | Open Source Project | HW    | Port | Endpoint             |
+| ------------ | ------------------- | ----- | ---- | -------------------- |
+| Embedding    | Langchain           | Gaudi | 6000 | /v1/embaddings       |
+| Retriever    | Langchain           | Xeon  | 7000 | /v1/retrieval        |
+| Reranking    | Langchain           | Xeon  | 8000 | /v1/reranking        |
```
Comment (louie-tsai) [inline ChatQnA/README.md:109]:
```suggestion
| Reranking    | Langchain, TEI          | Xeon  | 8000 | /v1/reranking        |
```

File: ChatQnA/README.md [hunk 5]
```diff
@@ -97,6 +97,21 @@ flowchart LR
...
 
+In the below, we provide a table that describes for each microservice component in the ChatQnA architecture, the default configuration of the open source project, hardware, port, and endpoint.
+
+<details>
+<summary> Gaudi default compose.yaml </summary>
+
+| MicroService | Open Source Project | HW    | Port | Endpoint             |
+| ------------ | ------------------- | ----- | ---- | -------------------- |
+| Embedding    | Langchain           | Gaudi | 6000 | /v1/embaddings       |
+| Retriever    | Langchain           | Xeon  | 7000 | /v1/retrieval        |
```
Comment (louie-tsai) [inline ChatQnA/README.md:108]:
```suggestion
| Retriever    | Langchain, Redis           | Xeon  | 7000 | /v1/retrieval        |
```

File: ChatQnA/README.md [hunk 6]
```diff
@@ -97,6 +97,21 @@ flowchart LR
 
 This ChatQnA use case performs RAG using LangChain, Redis VectorDB and Text Generation Inference on Intel Gaudi2 or Intel XEON Scalable Processors. The Intel Gaudi2 accelerator supports both training and inference for deep learning models in particular for LLMs. Visit [Habana AI products](https://habana.ai/products) for more details.
 
+In the below, we provide a table that describes for each microservice component in the ChatQnA architecture, the default configuration of the open source project, hardware, port, and endpoint.
+
+<details>
+<summary> Gaudi default compose.yaml </summary>
```
Comment (louie-tsai) [inline ChatQnA/README.md:103]:
```suggestion
<summary> **Gaudi default compose.yaml**</summary>
```

**Review by louie-tsai — COMMENTED** [review]
minor changes. hope to get all related open source projects in the table

File: ChatQnA/README.md [hunk 7]
```diff
@@ -97,6 +97,21 @@ flowchart LR
...
+
+| MicroService | Open Source Project | HW    | Port | Endpoint             |
+| ------------ | ------------------- | ----- | ---- | -------------------- |
+| Embedding    | Langchain           | Gaudi | 6000 | /v1/embaddings       |
```
[the remark this answers is not in the document]
Comment (ctao456) [reply ChatQnA/README.md]: Fixed in the updated table. Please review.

**Review by louie-tsai — APPROVED** [review]
looks good to me. thanks

**Review by yinghu5 — APPROVED** [review]
Looks good to me

**Pull request merged**